### ISSUE TITLE
Xcode 15 getHashAlgo

### ISSUE THREAD

**Issue by Kharchevskyi** [FlowCrypt/flowcrypt-ios#655]
func getHashAlgo(name: String) throws -> HashAlgo {

'CC_MD5' was deprecated in iOS 13.0: This function is cryptographically broken and should not be used in security contexts. Clients should migrate to SHA256 (or stronger).

**Comment by tomholub** [comment]
This is true. I'll have to see if we still have to maintain this MD5 for OpenPGP compatibility - it may be used somewhere in a non-dangerous manner.
